desmume/src/frontend/interface/interface.cpp

```diff
@@ -97,11 +98,23 @@ EXPORTED void desmume_set_language(u8 lang)
 EXPORTED int desmume_open(const char *filename)
 {
     int i;
+	if (rom_opened) {
```

Comment:
you seem to have used spaces for indent in the past, now tab

Reply:
Thanks, I normalized it for those lines. Though the file still contains whitespaces differences, but I think this is a more general issue.